### ISSUE TITLE
long time to load

### ISSUE THREAD

**Issue by ManoloVera** [files-community/Files#13341]
### Description

Hi, I am using Win 11 and all the time when i start using the app is taken around 1 or 2 min to be usable.
i see on the task manager the processor, memory all is normal and no idea why is happening.

### Steps To Reproduce

Start the Laptop
Open the app

### Requirements

- e.g. Fix crash that occurs when...

### Files Version

2.5.21.0

### Windows Version

Win 11 22H2 (OS Build 22621.2134)

### Log File

no issue log file

**Comment by jlopez788** [comment]
I've noticed this issue as well. Looks like the preview v2.5.29.0 opens fairly quick. Much quicker than the stable version

**Comment by jlopez788** [comment]
I've noticed this issue as well. Looks like the preview v2.5.29.0 opens fairly quick. Much quicker than the stable version

**Comment by ManoloVera** [comment]
thanks i am going to install this version and let you know

**Comment by swanicorn** [comment]
I had the same problem, except mine loaded annoyingly slow but not extreme like 1 or 2 minutes like the OP. I just tried the preview version 2.5.29.0. Preview version seems to have solved the issue. 

In Advanced settings, I noticed that it has a new setting like "Leave app running in background", which is absent in stable version 2.5.21.0. It is ON by default in Preview version. Maybe that solved the issue.
